packages/neos-ui-editors/src/Editors/Reference/referenceDataLoader.js

```diff
@@ -0,0 +1,134 @@
+import React, {PureComponent} from 'react';
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux';
+import {$get, $transform} from 'plow-js';
+import {selectors} from '@neos-project/neos-ui-redux-store';
+import {neos} from '@neos-project/neos-ui-decorators';
+
+export default isMulti => WrappedComponent => {
```

Comment:
Maybe wrap the `isMulti` bool into an object? Would be better readable when using the HOC and it's extendable in the future! :) :100: